### DISCUSSION TITLE
Can I recover data with minio and pulsar in standalone mode?

### DISCUSSION THREAD

**Discussion by byeong0** [milvus-io/milvus#21167] [Q&A and General discussion]
While scaling out the data node with Milvus, it became an abnormal state.
System View of attus is normal, but collections are not released or loaded. Also no search.
Can I recover data with minio and pulsar in standalone mode?

**Comment by yhmo** [comment]
The standalone use rocksmq to store DML logs, just like the Pulsar for cluster. I think it is not possible to do that.

**Reply by byeong0** [reply]
Can I recover collections with just the data from pulsar?

**Reply by xiaofan-luan** [reply]
> Can I recover collections with just the data from pulsar?

It could be doable, we tried to use etcd, pulsar and minio on a single pvc and recover it works.
If you can open an issue and offer server side logs and we can probably take a deep look into it. 
One easiest way is to use birdwatcher and force release the stucked collection
